docs/changes.rst

```diff
@@ -133,10 +133,18 @@ Version 1.1.0b6 (unreleased)
 
 Compatibility notes:
 
-- Repositories in a repokey mode with a blank passphrase are now treated
-  as unencrypted repositories for security checks
+- Repositories in a repokey mode (including "authenticated" mode) with a
```

Comment:
that phrasing is a bit problematic.

i see what you mean with "a repokey mode" as a more general term than specifically the one "-E repokey" mode, but a normal user will not see / know that.

so better just say "Repositories in 'repokey' (or 'authenticated') mode with an empty passphrase ..."

(also note "empty" is maybe more clear than "blank"? see also text below.)